java/robox-slicer-extension/robox-slicer-flow/src/main/java/com/roboxing/slicerextension/flow/Slicer.java

```diff
@@ -180,7 +182,7 @@ public void postProcess(File inputGCode, File resultGCode) throws IOException{
                 }
                 if ((m = java.util.regex.Pattern.compile("^(M204\\s+S(\\d+))").matcher(strLine)).find()) {
                     // printf NEW "M201 X%d Y%d Z%d E2000\n", $2, $2, $2;
-                    writeOutput(String.format("M201 X%d Y%d Z%d E2000\n", m.group(2), m.group(2), m.group(2)));
+                    writeOutput(String.format("M201 X%d Y%d Z%d E2000" + newLine, m.group(2), m.group(2), m.group(2)));
```

Comment:
Don't forget to replace write with printlns (without + newLine) wherever we have PrintWriter above them!